src/util/std_expr.h

```diff
@@ -2126,7 +2126,6 @@ inline not_exprt &to_not_expr(exprt &expr)
   return ret;
 }
 
-
```

Comment:
Clean ups should be on separate commit.

Reply:
I'd say best to simply remove this change, and not aim for this minor cleanup in this PR at all.

Reply:
Thanks for spotting this! I removed this empty line by mistake.